src/main/scala/com/fulcrumgenomics/bam/FindTechnicalReads.scala

```diff
@@ -47,8 +48,12 @@ import scala.collection.JavaConversions.asScalaIterator
      |technical sequences.  Sub-sequences are generated from both the sequences and the
      |reverse complement of the sequences, ignoring any sub-sequences that include 'N's.
      |
-     |The output BAM file will contain all reads that matched to a sub-sequence of the
-     |technical sequences and, if the read is paired, the read's mate pair.
+     |By default the output BAM file will contain all reads that matched to a sub-sequence of the
+     |technical sequences and, if the read is paired, the read's mate pair.  An option is
+     |available to apply a tag to matched reads (--tag/-t), and if specified each matching
+     |read will be tagged with the 0-based index of the sequence to which it matched. In
```

Comment:
That's going to make everything much slower as it would have to do an exhaustive search instead of aborting at the first kmer that matched.